janitor/db/database.py

```diff
@@ -81,17 +88,17 @@ def write_entries_to_table(
             entries {Sequence[Mapping[str, Any]]}: list of parsed entries to add to table
             rows_per_query {int}: number of rows per batch
         """
+        num_entries = len(entries)
+        index = 0
+
         try:
-            self.connection.start_transaction()
-            num_entries = len(entries)
-            entries_index = 0
-
-            while entries_index < num_entries:
-                self.cursor.executemany(
-                    query,
-                    list_of_entries_values(entries[entries_index : entries_index + rows_per_query]),  # noqa
-                )
-                entries_index += rows_per_query
+            with self.connection.cursor() as cursor:
+                self.connection.start_transaction()
```

Comment:
Maybe this should be outside the `with` block unless it needs to be in here.  I'm just concerned that the transaction is started inside the `with` block and the `commit()` is called outside the block.